database/migrations/2019_02_23_222617_add_hidpi.php

```diff
@@ -17,55 +17,48 @@ class AddHidpi extends Migration
 	 */
 	public function up()
 	{
-		if (!Schema::hasColumn('photos', 'thumb2x')) {
-			if (Schema::hasTable('configs')) {
-
-				DB::table('configs')->insert([
-					[
-						'key'   => 'thumb_2x',
-						'value' => '1'
-					],
-					[
-						'key'   => 'small_2x',
-						'value' => '0'
-					],
-					[
-						'key'   => 'medium_2x',
-						'value' => '0'
-					],
-				]);
-			}
-			else {
-				echo "Table configs does not exists\n";
-			}
-
-			if (Schema::hasTable('photos')) {
-				Schema::table('photos', function (Blueprint $table) {
-					$table->renameColumn('medium', 'medium_old');
-				});
-
-				Schema::table('photos', function (Blueprint $table) {
-					$table->renameColumn('small', 'small_old');
-				});
-
-				Schema::table('photos', function (Blueprint $table) {
-					$table->char('medium', 20)->default('');
-					$table->char('medium2x', 20)->default('');
-					$table->char('small', 20)->default('');
-					$table->char('small2x', 20)->default('');
-					$table->boolean('thumb2x')->default(true);
-				});
-			}
+		if (Schema::hasTable('configs')) {
```

Comment:
Can you please add back the check for 
```php
if (!Schema::hasColumn('photos', 'thumb2x')) {
```
here ? :)
it is required so the migration can be rerun in case of that specific one fails.

Reply:
Fixed